plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1051,10 +1062,13 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
 
       override fun onAnimationStart(animation: Animator) {}
 
-      override fun onAnimationCancel(animation: Animator) {}
+      override fun onAnimationCancel(animation: Animator) {
+        cameraAnimationsPlugin.anchor = scaleCachedAnchor
+        dispatchCameraIdle()
```

Comment:
Isn't `onAnimationEnd` always following `onAnimationCancel`? Do we need to duplicate the logic here?

Reply:
https://github.com/mapbox/mapbox-maps-android/pull/280